backend/spec/features/sample_capybara_spec.rb

```diff
@@ -1,6 +1,42 @@
 describe 'ToDo Lists', driver: :apparition do
-  it 'opens app page' do
+  toDoList = ['Cypress', 'Playwright', 'Capybara', 'Watir']
+  before(:each) do
     visit 'http://localhost:3030'
-    expect(page).to have_content 'Add'
   end
+
+  after(:each) do
+    click_button('Click me')
+  end
+
+  it 'adds a new list' do
+    toDoList.each do |list|
+    fill_in 'New list', with: list
+    find(:xpath, "//div[@class='Header']//button[@type='button'][normalize-space()='Add']").click
+    end
+    number_of_list = find(:css, 'div#root button[type="button"]:nth-child(2)').text
+    expect(number_of_list).to eq("Number of lists: #{toDoList.length}")
+    toDoList.each do |list|
+    expect(page).to have_text(list)
+    end
+  end
+  
+  it 'A List Item status can be changed by clicking the associated checkbox' do
+    fill_in 'New list', with: "Capybara"
+    find(:xpath, "//div[@class='Header']//button[@type='button'][normalize-space()='Add']").click
+    fill_in 'New item', with: "locating elements"
+    find(:css, 'div#root div.list-group.list-group-flush > div > div > div > button[type="button"]').click
```

Comment:
The same here. I we can't add a data-test label in front-end code for some reason then I would better rely on the button name rather than such a complex css